Added config (#931)
memgraph/documentation: pages/clustering/high-availability.mdx

Current deployment assumes the existence of only one datacenter, which automatically means that Memgraph won't be available in the case the whole datacenter goes down. We are actively
working on 2 datacenter (2-DC) architecture.

## Raft configuration parameters

> There are several Raft-related parameters which are important for correct functioning of the cluster. First, leader coordinator sends heartbeat

Reviewer: ```suggestion
Several Raft-related parameters are important for the correct functioning of the cluster.  The leader coordinator sends a heartbeat
```

message to other coordinators every second to determine their health. This configuration option is connected with leader election timeout which
is a randomized value from the interval [2000ms, 4000ms] and which is used by followers to decide when to trigger new election process. Leadership

> expiration is set to 1800ms so that cluster can never get into situation where multiple leaders exist. These concrete values give a cluster

Reviewer: ```suggestion
expiration is set to 1800ms so that cluster can never get into situation where multiple leaders exist. These specific values give a cluster
```
